.travis.yml

```diff
@@ -11,4 +11,4 @@ install:
 before_script:
   - echo "MAVEN_OPTS=${TEST_MEM_SETTINGS}" > ~/.mavenrc
 script:
-  - mvn jacoco:report coveralls:report --batch-mode -Dtest.arguments="${TEST_MEM_SETTINGS}"
+  - mvn package jacoco:report coveralls:report --batch-mode --activate-profiles coveralls -Dtest.arguments="${TEST_MEM_SETTINGS}"
```

Comment:
Sorry, partly my mistake in my merge, I intended to move coveralls over to the GH action and not in Travis so we don't get two reports. Can you please remove `coveralls:report` and `--activate-profiles coveralls` from here. `package` and `jacoco` are enough.